src/main/resources/project-defaults.yml

```diff
@@ -5,7 +5,7 @@ swarm:
     data-sources:
       defaultDB:
         user-name: "sa"
-        connection-url: "jdbc:h2:tcp://localhost:1521/test"
+        connection-url: "jdbc:h2:tcp://localhost:1818/test"
```

Comment:
Attention port 1818 works for me, because 1521 doesn't work...

Reply:
let the config untouched. use another config file for your development.